portfolio/src/main/webapp/script.js

```diff
@@ -15,20 +15,40 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Arms for your chairs are just chairs for your arms.', 'I own chickens', 'sup and wassup', 'yeet'];
+// function addRandomGreeting() {
+//   const greetings =
+//       ['Arms for your chairs are just chairs for your arms.', 'I own chickens', 'sup and wassup', 'yeet'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+//   // Pick a random greeting.
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+//   // Add it to the page.
+//   const greetingContainer = document.getElementById('greeting-container');
+//   greetingContainer.innerText = greeting;
+// }
+
+// function getHello() {
+//     fetch('/data').then(response => response.text()).then((greeting) => {
+//     document.getElementById('greeting-container').innerText = greeting;
+//   });
+// }
+
+function getComments() {
+    fetch('/data').then(response => repsonse.json()).then((comment) => {
```

Comment:
Misspelling. Check that your code works.

Reply:
Somehow it still worked! i fixed it right now it prints
{"comment1": "sup Autumn", "comment2": "What other projects have you done", "comment3": "hi"}